### PR TITLE
Clean up heron-client-install.sh

### PR DESCRIPTION
**Pull request by nwangtw**
heron-client-install.sh and heron-tools-install.sh have been merged into heron-install.sh, but scripts/documents haven't been updated accordingly yet.

### CODE REVIEW

**Comment by xiaoyao1991** [comment]
👍

File: docker/scripts/ci-docker.sh [hunk 1]
```diff
@@ -66,18 +66,14 @@ build_exec_image() {
   setup_scratch_dir $SCRATCH_DIR
 
   # need to copy artifacts locally
-  TOOLS_FILE="$OUTPUT_DIRECTORY/heron-tools-install.sh"
-  TOOLS_OUT_FILE="$SCRATCH_DIR/artifacts/heron-tools-install.sh"
+  TOOLS_FILE="$OUTPUT_DIRECTORY/heron-install.sh"
```
Comment (nlu90) [inline docker/scripts/ci-docker.sh]: variable name`INSTALL_FILE` makes more sense. And also same the followings

File: scripts/applatix/test.sh [hunk 2]
```diff
@@ -36,16 +36,10 @@ JAVA_INTEGRATION_TESTS_BIN="${HOME}/.herontests/lib/integration-tests.jar"
 PYTHON_INTEGRATION_TESTS_BIN="${HOME}/.herontests/lib/heron_integ_topology.pex"
 SCALA_INTEGRATION_TESTS_BIN="${HOME}/.herontests/lib/scala-integration-tests.jar"
 
-# install client
-T="heron client install"
-start_timer "$T"
-python ${UTILS}/save-logs.py "heron_client_install.txt" ./heron-client-install.sh --user
-end_timer "$T"
-
 # install tools
```
Comment (nlu90) [inline scripts/applatix/test.sh]: install clients and tools

File: scripts/applatix/testutils.sh [hunk 3]
```diff
@@ -31,16 +31,10 @@ echo "Using $PLATFORM platform"
...
 export PATH=${HOME}/bin:$PATH
 
-# install client
-T="heron client install"
-start_timer "$T"
-python ${UTILS}/save-logs.py "heron_client_install.txt" ./heron-client-install.sh --user
-end_timer "$T"
-
 # install tools
 T="heron tools install"
```
Comment (nlu90) [inline scripts/applatix/testutils.sh]: client and tools

File: docker/scripts/ci-docker.sh (hunk 1 above)
Comment (nwangtw) [reply docker/scripts/ci-docker.sh]: cool. thx.

**Pull request merged**